shadows/framework/src/main/java/org/robolectric/shadows/ShadowParcelFileDescriptor.java

```diff
@@ -142,8 +147,13 @@ protected int getFd() {
 
   @Implementation
   protected void close() throws IOException {
+    if (closed) {
```

Comment:
The method called `closeWithStatus` called by `ParcelFileDescriptor#close` uses `mClosed` field to judge whether it needs to execute real close operation. It's correct, but could you add some comments about it? Let later developers know why you use `closed` at the begin of `close` method.

Reply:
https://android.googlesource.com/platform/frameworks/base/+/refs/tags/android-4.4_r1/core/java/android/os/ParcelFileDescriptor.java#634

Reply:
OK, I think this explicit check & return has no side effect, but it does need a comment.